versions.gradle

```diff
@@ -7,7 +7,7 @@
 
 // casual-java group and version
 group = 'se.laz.casual'
-version = '3.2.50'
+version = '3.2.51'
```

Comment:
Make this a version `3.3.0` to reflect the change we have made to the behaviour.